src/editor-mathfield/render.ts

```diff
@@ -35,10 +35,15 @@ export function requestUpdate(
   if (!mathfield.dirty) {
     mathfield.dirty = true;
     requestAnimationFrame(() => {
-      if (isValidMathfield(mathfield) && mathfield.dirty) {
+      if (
+        isValidMathfield(mathfield) &&
+        mathfield.dirty &&
+        !mathfield.options.readOnly
+      ) {
         mathfield._atomBoundsCache = new Map<string, Rect>();
         render(mathfield, options);
         mathfield._atomBoundsCache = undefined;
+        mathfield.attachNestedMathfield();
```

Comment:
Is this right? It does the attach only if the mathfield is *not* readonly?

Reply:
you're right.